twoliter/src/lock.rs

```diff
@@ -243,20 +390,21 @@ impl Lock {
                     "image",
                     "inspect",
                     image_uri.as_str(),
-                    "--format \"{{ json .Config.Labels }}\"",
+                    "--format",
+                    "\"{{ json .Config.Labels }}\"",
                 ],
                 format!(
                     "failed to fetch kit metadata for {} with digest {}",
                     image.to_string(),
                     manifest.digest
                 )
             );
+            let label_str = String::from_utf8_lossy(label_bytes.as_slice()).to_string();
+            let label_str = label_str.trim().trim_matches('"');
             // Otherwise we should have a list of json blobs we can fetch the metadata from the label
-            let labels: HashMap<String, String> = serde_json::from_slice(label_bytes.as_slice())
-                .context(format!(
-                    "could not deserialize labels on the image for {}",
-                    image
-                ))?;
+            let labels: HashMap<String, String> = serde_json::from_str(label_str).context(
+                format!("could not deserialize labels on the image for {}", image),
+            )?;
```

Comment:
What does the "otherwise" comment above refer to? Is there some other source of kit metadata?

Reply:
Wasn't very helpful comment, removed it